docs/api.rst

```diff
@@ -1,77 +1,119 @@
 Interfaces, exceptions and utils
 ================================
 
-Connection Clients
-------------------
+The naming of things can be confusing when using Bluetooth LE, due to the many possible meanings of
+words like "device" or "service" depending on the context. Here is a quick breakdown of the main object
+types used in bleak and their meaning, roughly in the order in which you will often use them in your code:
 
-Interface
-~~~~~~~~~
+* :py:class:`bleak.BleakScanner` finds advertising BLE devices
+* :py:class:`bleak.backends.device.BLEDevice` the representation (think: name or address) of such a BLE device
+* :py:class:`bleak.BleakClient` an open connection to a BLE device
+* :py:class:`bleak.backends.service.BleakGATTService` the name (or address) of a service implemented in a BLE device
+* :py:class:`bleak.backends.service.BleakGATTCharacteristic` the name (or address) of a charactieristic (think: attribute or variable) implemented by such a service
 
-.. automodule:: bleak.backends.client
+You can then read and write characteristic values by calling methods of :py:class:`bleak.backends.device.BLEDevice`,
+passing the :py:class:`bleak.backends.service.BleakGATTCharacteristic` of the value you are interested in.
+
+Connecting
+----------
+
+Usually you will use the top-level :py:class:`bleak.BleakClient` object to connect to a BLE 
+device annd communicate with it. This class inherits the correct implementation 
+for your platform. Some of the methods, especially the constructor, may have 
+additional platform-specific arguments which you can find in the relevant 
+implementation section.
+
+
+Client interface
+~~~~~~~~~~~~~~~~
+
+.. autoclass:: bleak.BleakClient
     :members:
+    :inherited-members:
 
-Windows
-~~~~~~~
+Client Windows implementation
```

Comment:
It would be nice to move the backend docs to a separate page.

Reply:
Done.